### ISSUE TITLE
SMTP without TLS doesn't work

### ISSUE THREAD

**Issue by 7ussana** [vmware/harbor#4268]
Hello,

I have installed Harbor but sending mail via an no authenticated internal smtp server doesn't work.

The ping is ok for information 
I have this in the **ui.log**

```
Feb 13 14:37:41 172.19.0.1 ui[447]: 2018-02-13T13:37:41Z [DEBUG] [mail.go:92]: establishing TCP connection with smtp.******.com:25 ...
Feb 13 14:37:41 172.19.0.1 ui[447]: 2018-02-13T13:37:41Z [DEBUG] [mail.go:117]: creating SMTP client for smtp.*********.com ...
Feb 13 14:37:41 172.19.0.1 ui[447]: 2018-02-13T13:37:41Z [DEBUG] [mail.go:134]: the email server smtp.********.com:25 does not support STARTTLS
Feb 13 14:37:41 172.19.0.1 ui[447]: 2018-02-13T13:37:41Z [DEBUG] [mail.go:139]: authenticating the client...
Feb 13 14:37:41 172.19.0.1 ui[447]: 2018-02-13T13:37:41Z [ERROR] [email.go:109]: failed to ping email server: unencrypted connection

```
My **harbor.cfg**

```
email_identity =

email_server = smtp.******.com
email_server_port = 25
email_username =
email_password =
email_from = Admin Docker-Hub  <docker-hub_admin@*****.com>
email_ssl = false
email_insecure = true
```

**Docker version**

```
Server:
 Engine:
  Version:  17.12.0-ce
  API version: 1.35 (minimum version 1.12)
  Go version: go1.9.2
  Git commit: c97c6d6
  Built: Wed Dec 27 20:09:54 2017
  OS/Arch:  linux/amd64
  Experimental: false
```

**Docker Compose version**

```
docker-compose version 1.8.0, build unknown
docker-py version: 1.9.0
CPython version: 2.7.13
OpenSSL version: OpenSSL 1.1.0f  25 May 2017
```

**Comment by ywk253100** [comment]
The credentials are only sent when the connection is using TLS for security issue. So make sure your SMTP server supports TLS.

**Comment by 7ussana** [comment]
I don't use a SMTP TLS server.

**Comment by Ducatel** [comment]
I have the exact same issue.
I use an internal mail server without TLS

**Comment by ywk253100** [comment]
Fixed by #4375.
